Doc/whatsnew/3.14.rst

```diff
@@ -367,6 +367,13 @@ errno
   (Contributed by James Roy in :gh:`126585`.)
 
 
+faulthandler
+------------
+
+* Add support for printing the C stack trace on systems that support it via
+  :func:`faulthandler.dump_c_stack` or via the *c_stack* argument in :func:`faulthandler.enable`.
```

Comment:
Don't forget to add (Contributed by XXX in :gh:\`...\`) to get the issue number.